### DISCUSSION TITLE
Compare to onnx, the performance is not OK.

### DISCUSSION THREAD

**Discussion by StephensCheng** [intel/openvino-plugins-ai-audacity#245] [General]
The Deepfilter provide the C api with onnx model, I have try it in the Mac pro M1 to process the audio file with background noise. The performance is OK. The CPU will reach into 99%. But with model openvino, the CPU will reach to 700%.

**Comment by RyanMetcalfeInt8** [comment]
It could be that with native onnx API / runtime, it's a bit more optimized for use with Mac pro M1. Unfortunately I don't have a Mac to try this.
